Doc/library/traceback.rst

```diff
@@ -372,7 +378,7 @@ exception and traceback:
        print("*** print_tb:")
        traceback.print_tb(exc_traceback, limit=1, file=sys.stdout)
        print("*** print_exception:")
-       traceback.print_exception(exc_type, exc_value, exc_traceback,
+       traceback.print_exception("ignored parameter", exc_value, exc_traceback,
```

Comment:
This does not seem ideal. It is less compatible with older Python versions. If it needs clarifying at all, perhaps a comment would be better?